### PR TITLE
Fixed setting cache for HTMLPurifier_Lexer to runtime folder instead of vendor by default

### PR DESCRIPTION
**Pull request by webdevsega**

### CODE REVIEW

File: framework/helpers/BaseStringHelper.php [hunk 1]
```diff
@@ -152,6 +152,7 @@ public static function truncateWords($string, $count, $suffix = '...', $asHtml =
     protected static function truncateHtml($string, $count, $suffix, $encoding = false)
     {
         $config = \HTMLPurifier_Config::create(null);
+        $config->set('Cache.SerializerPath', \Yii::$app->getRuntimePath());
```
Comment (samdark) [inline framework/helpers/BaseStringHelper.php]: Is it creating a subdir?

**Comment by samdark** [comment]
Overall a good change.

File: framework/helpers/BaseStringHelper.php (hunk 1 above)
Comment (webdevsega) [inline framework/helpers/BaseStringHelper.php]: Yes, HTMLPurifier_DefinitionCache_Serializer::_prepareDir create subdir 'HTML'.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**